src/components/organisms/footer/footer.html

```diff
@@ -0,0 +1,52 @@
+<div class="footer">
```

Comment:
```suggestion
<footer class="footer">
```
セマンティック要素を使える時はなるべく使っていきましょう！
https://developer.mozilla.org/ja/docs/Web/HTML/Element/footer